docs/source/concepts/architectural-model.rst

```diff
@@ -0,0 +1,284 @@
+  Each endpoint is connected to a subset of the **domains**.
+
+* Endpoints host parties on a subset of the domains they are connected to.
+  An endpoint's services provide a party with projected commits from the domains on which it hosts the party.
+
+
+Examples
+========
+
+In :ref:`global state topologies <global-state-topologies>`, at least one system contains a physical copy of the entire virtual shared ledger.
+Accordingly, these topologies need only one domain for the virtual shared ledger, namely such a system.
+In a :ref:`fully centralized ledger <fully-centralized-ledger>`, there is by definition only one such system.
+Accordingly, this system is the single domain for the virtual shared ledger.
+So the endpoint contains the physical ledger.
+
+.. image:: ./images/endpoint-with-physical-ledger.svg
+   :align: center
+   :width: 60%
+
+A single domain also suffices for :ref:`horizontally scaled <scaling-daml-ledgers>` and :ref:`replicated setups <daml-ledger-replication>`.
+Replication and concensus ensures that all writer nodes agree on the contents of the virtual shared ledger.
+Conceptually, they form a single domain even though endpoints may connect only to a subset of the writer nodes.
+
+Multiple domains appear with interoperability in :ref:`partitioned ledger topologies <partitioned-topologies>`.
+Interoperability makes it possible that an atomic transaction uses the output contracts of transactions that have been recorded to different deployed ledgers.
+For example, when an organization initially deploys two workflows to two ledgers, it can later compose those workflows into a larger workflow that spans the deployed ledger.
+In this setup, each deployed ledger constitutes a domain for the virtual shared ledger.
+Multiple domains can also be used to improve horizontal scaling of the atomic commit protocol.
+Endpoints participate only in the commit protocols of the domains they are connected to and the commit protocols of different domains can often run in parallel.
+The next diagram shows a such situation where party `A` is hosted on two endpoints `P1` and `P2` and party `B` is hosted on `P3`.
+The virtual shared ledger is fed from three domains.
+The arrows indicate that `P1` hosts `A` on domains 1 and 3 and `P2` on domains 1 and 2 and `P3` hosts `B` on all three.
+
+.. figure:: ./images/multiple-domains.svg
+   :align: center
+   :name: multiple-domains
+
+   Virtual shared ledger topology with three domains
+
+A view on the virtual shared ledger
+***********************************
+
+When an endpoint hosts a party only on a subset of the domains for the vitual shared ledger,
+then the endpoint's transaction and active contract services are derived only from commits from those domains.
+An endpoint therefore provides each hosted party with a **view** on their projection of the virtual shared ledger.
+The view consists of all the commits that originate from the domains on which the endpoint hosts the party.
```

Comment:
This could be interpreted to give a guarantee that all Participants know all the history of a party on the domain. Maybe rephrase to make clear that you get ACS and new transactions for hosted parties.